daprdocs/content/en/developing-applications/building-blocks/secrets/howto-secrets.md

```diff
@@ -28,146 +34,253 @@ Create a directory for your components file named `components` and inside it cre
 apiVersion: dapr.io/v1alpha1
 kind: Component
 metadata:
-  name: my-secrets-store
+  name: localsecretstore
   namespace: default
 spec:
   type: secretstores.local.file
   version: v1
   metadata:
   - name: secretsFile
-    value: <PATH TO SECRETS FILE>/mysecrets.json
+    value: secrets.json
   - name: nestedSeparator
     value: ":"
 ```
 
-Make sure to replace `<PATH TO SECRETS FILE>` with the path to the JSON file you just created.
-
->Note: the path to the secret store JSON is relative to where you call `dapr run` from.
-
-
 To configure a different kind of secret store see the guidance on [how to configure a secret store]({{<ref setup-secret-store>}}) and review [supported secret stores]({{<ref supported-secret-stores >}}) to see specific details required for different secret store solutions.
 ## Get a secret
 
 Now run the Dapr sidecar (with no application)
```

Comment:
This line conflicts with the examples shown as each runs dapr with an application.